app/views/LocationShare/LocationPreviewModal.tsx

```
@@ -0,0 +1,172 @@
+	const mounted = useRef(true);
+	useEffect(
+		() => () => {
+			mounted.current = false;
+		},
+		[]
+	);
+	const safeSet = (fn: () => void) => {
+		if (mounted.current) fn();
+	};
+
+	const { id, token, username } = useAppSelector(
```

ESLint

[error] 'token' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)